requirements/requirements.txt

```diff
@@ -364,7 +364,6 @@ six==1.14.0
     #   pyrsistent
     #   python-dateutil
     #   tacacs-plus
-    #   twilio
```

Comment:
This appears to be completely legit.

The library really did remove its dependency on six in https://github.com/twilio/twilio-python/commit/e2df36ac5141ce5b540b2de72a520cd2b39e81cd